Sources/SwiftDate/DateInRegionFormatter.swift

```diff
@@ -327,6 +330,65 @@ public class DateInRegionFormatter {
 		localized_str = String(format: localized_str, arguments: arguments)
 		return localized_str
 	}
+    
+    /// Return true if date for russian locale, number are plural and localized strings has right translate for this unit
+    ///
+    /// - parameter unit:     unit of calendar component
+    /// - parameter value:    the value associated with the unit
+    ///
+    /// - returns: true or false
+    private func isRussianSpecifyCase(unit: Calendar.Component, withValue value: Int) -> Bool {
+        if locale?.identifier.hasPrefix("ru") == false {
+            return false
+        }
+        
+        if value <= 1 {
+            return false
+        }
+        
+        let availableUnits: [Calendar.Component] = [.minute, .hour, .day, .weekOfYear, .month]
+        if availableUnits.contains(unit) == false {
+            return false
+        }
+        
+        return true
+    }
+    
+    /// Calculate right ending type (RussianPluralEnding) of plural using russian language's rules
+    ///
+    /// - parameter value:    the value associated with the unit
+    ///
+    /// - returns: additional string for russian localize key
+    private func russianPluralEnding(value: Int) -> String {
+        var reminder = value % 100
+        var result: String
+        if reminder <= 10 || reminder >= 20 {
+            reminder = reminder % 10
+            switch reminder {
+            case 1:
+                return RussianPluralEnding.one.rawValue
+            case 2...4:
+                return RussianPluralEnding.fromTwoToFour.rawValue
+            default:
+                break;
+            }
+        }
+        
+        return RussianPluralEnding.defaultCase.rawValue
+    }
+    
+    // In English we create a plural by simply adding the letter 's'. As the Russian language is based on the case system there are different plurals in each case.
```

Comment:
~`As`~ `However,` the Russian language is based on `a` case system `where` there are...